test/suite/test_hs24.py

```diff
@@ -36,15 +37,22 @@ class test_hs24(wttest.WiredTigerTestCase):
     session_config = 'isolation=snapshot'
     uri = 'table:test_hs24'
 
+    key_format_values = [
+        ('column', dict(key_format='r')),
+        ('integer_row', dict(key_format='i')),
+    ]
+
+    scenarios = make_scenarios(key_format_values)
+
     value1 = 'a' * 500
     value2 = 'b' * 500
     value3 = 'c' * 500
     value4 = 'd' * 500
     def test_zero_ts(self):
-        self.session.create(self.uri, 'key_format=i,value_format=S')
+        self.session.create(self.uri, 'key_format={},value_format=S'. format(self.key_format))
         self.conn.set_timestamp('oldest_timestamp=' + self.timestamp_str(1))
         cursor = self.session.open_cursor(self.uri)
-        for i in range(0, 2000):
```

Comment:
Instead of a magic number `2000` how about using `nrows = 2000` and use the variable here?

Reply:
fixed, thanks :-)